src/javascript/_common/third_party_links.js

```diff
@@ -0,0 +1,50 @@
+const State            = require('./storage').State;
+const Client           = require('../app/base/client');
+const BinarySocket     = require('../app/base/socket');
+const Dialog           = require('../app/common/attach_dom/dialog');
+
+const ThirdPartyLinks = (() => {
+    const init = () => {
+        if (Client.isLoggedIn()) {
+            BinarySocket.wait('landing_company').then(() => {
+                if (State.getResponse('landing_company.financial_company.shortcode') === 'maltainvest') {
+                    document.body.addEventListener('click', clickHandler);
+                }
+            });
+        }
+    };
+
+    const clickHandler = (e) => {
+        if (!e.target) return;
+        const link_el = e.target.closest('a');
+        if (!link_el) return;
+
+        const dialog = document.querySelector('#third_party_redirect_dialog');
+        if (dialog && dialog.contains(link_el)) return;
+
+        if (isThirdPartyLink(link_el.href)) {
+            e.preventDefault();
+            Dialog.confirm({
+                id     : 'third_party_redirect_dialog',
+                // TODO: pass array of sentences once negar/show_malta_popup_mt card is merged
+                message: 'You will be redirected to a third-party website which is not owned by Binary.com. Click OK to proceed.',
+            }).then((should_proceed) => {
+                if (should_proceed) window.open(link_el.href, '_blank');
```

Comment:
there was a vulnerability with `_blank`, i think we should add `rel="noopener noreferrer"` attribute to the link as well

Reply:
I'm confused, this is just opening the link in a new tab by javascript command, it's not setting blank on a link, how will we pass a noopener noreferrer to this?

Reply:
ohh right, that's for `<a>` tag. for `window.open()` method, you can set the opener property to null; e.g.:
```
let link = window.open();
link.opener = null;
```